cmake/partition_manager.cmake

```diff
@@ -13,7 +13,11 @@ Also, the each image's hex file will be automatically associated with its partit
 get_property(PM_IMAGES GLOBAL PROPERTY PM_IMAGES)
 get_property(PM_SUBSYS_PREPROCESSED GLOBAL PROPERTY PM_SUBSYS_PREPROCESSED)
 
-set(static_configuration_file ${APPLICATION_SOURCE_DIR}/pm_static.yml)
+if(EXISTS "${pm_static_CONF_FILE}" AND NOT IS_DIRECTORY "${pm_static_CONF_FILE}")
+  set(static_configuration_file ${pm_static_CONF_FILE})
+else()
+  set(static_configuration_file ${APPLICATION_SOURCE_DIR}/pm_static.yml)
+endif()
```

Comment:
We use CONF_FILE for something else.

What about PM_STATIC_YML_FILE ?

Reply:
I thought about re-using it to point pin it as a configuration file. But I had doubts myself if this will not be confused with Kconfig files.
So variable renamed.